### PR TITLE
fix: improve errors and messages

### LINKED ISSUE #35: No API key has been provided error, even after successful login
image

### PR DESCRIPTION
**Pull request by cyyynthia**
closes #35 - solves the confusion by saying the Project ID is missing. Keys are now shown in alphabetical order and grouped by namespace.

### CODE REVIEW

File: src/commands/sync/compare.ts [hunk 1]
```diff
@@ -39,19 +39,21 @@ async function compareHandler(this: Command, pattern: string) {
 
   console.log('Your code project and Tolgee project are out of sync.');
   if (diff.added.length) {
-    console.log(ansi.green.bold(`${diff.added.length} new strings`));
+    const key = diff.added.length === 1 ? 'key' : 'keys'
+    console.log(ansi.green.bold(`${diff.added.length} new ${key} found in your code project`));
```
Comment (stepan662) [inline src/commands/sync/compare.ts]: the "in your code project" doesn't sound right for me, it should be "in your project code", however I think it's maybe unnecessary to write it at all "10 new keys found" is understandable, I think

File: src/commands/sync/compare.ts [hunk 2]
```diff
@@ -39,19 +39,21 @@ async function compareHandler(this: Command, pattern: string) {
...
     }
 
     // Line break
     console.log('');
   }
 
   if (diff.removed.length) {
-    console.log(ansi.red.bold(`${diff.removed.length} removed strings`));
+    const key = diff.added.length === 1 ? 'key' : 'keys'
+    console.log(ansi.red.bold(`${diff.removed.length} ${key} exist on Tolgee but were not found in your code project`));
```
Comment (stepan662) [inline src/commands/sync/compare.ts]: Same here, it could be "10 unused keys"

File: src/commands/sync/compare.ts (hunk 1 above)
Comment (cyyynthia) [reply src/commands/sync/compare.ts]: I use "code project" in contrast to the "localization project" on Tolgee. I guess the short version works here.

File: src/commands/sync/compare.ts (hunk 2 above)
Comment (cyyynthia) [reply src/commands/sync/compare.ts]: For this one I'm unsure, unused may be weird to inexperienced users and having the CLI clearly tell things might help. But maybe I'm making it too foolproof here 😅 

I'll go back to the simple way here too I guess.

**Pull request merged**